src/Compilers/Core/Portable/Operations/IIsNullOperation.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    /// <summary>
+    /// Represents result of checking whether the <see cref="Operand"/> is null. 
+    /// For reference types this checks if the <see cref="Operand"/> is a null reference, 
+    /// for nullable types this checks if the <see cref="Operand"/> doesn’t have a value.
+    /// The node is only produced as part of a flow graph during rewrite of <see cref="ICoalesceOperation"/> and 
+    /// <see cref="IConditionalAccessOperation"/> nodes.
```

Comment:
💭 This last sentence is concerning to me. It stands out as a statement which is particularly susceptible to becoming incorrect over time. When facing a statement like this, I would challenge the author to attempt to rewrite the statement using more maintainable language per the following:

1. Does this statement provide new information likely to influence the manner in which an API consumer interacts with the interface? If this is 'no' for all scenarios, the sentence is likely removable, which is the most maintainable outcome.
2. Are any constraints stated in the sentence unnecessarily restrictive? The two constraints involved here are:
    1. The node is only produced as part of a flow graph. One example of a relaxation of this constraint would be restricting the creation of this node to post-processing operations on an `IOperation` tree, or otherwise stating that basic `IOperation` analyzers will not encounter this node.
    2. The node is only produced from `ICoalesceOperation` and/or `IConditionalAccessOperation`. This constraint could be relaxed either by moving from an inclusion list to an exclusion list, or by modifying the wording to treat these as examples instead of all-inclusive. (This constraint is particularly important to address due to its impact on future versions of the language.)

Reply:
I think it is useful information that it is produced for that purpose. I think it is not a good idea to specify that it is *only* produced for that purpose. I suggest removing the word "only".